optaplanner-examples/pom.xml

```diff
@@ -210,6 +211,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+    </dependency>
```

Comment:
`LocalDate` can not be serialized by default.

Reply:
I haven't noticed any extra annotation coming from this dependency; isn't it rather a `runtime` dependency? Perhaps we would not need to ignore this dependency as unused if it had the `<scope>runtime</runtime>`.